[2023-12-27 17:54:25,262] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.extraer_data manual__2023-12-27T17:54:20.503605+00:00 [queued]>
[2023-12-27 17:54:25,299] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.extraer_data manual__2023-12-27T17:54:20.503605+00:00 [queued]>
[2023-12-27 17:54:25,300] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 17:54:25,301] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-27 17:54:25,302] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 17:54:25,342] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extraer_data> on 2023-12-27 17:54:20.503605+00:00
[2023-12-27 17:54:25,356] {standard_task_runner.py:52} INFO - Started process 13058 to run task
[2023-12-27 17:54:25,372] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'OpenWeather_ETL2', 'extraer_data', 'manual__2023-12-27T17:54:20.503605+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/ETL_OpenWeather2.py', '--cfg-path', '/tmp/tmp_wk1xjcu', '--error-file', '/tmp/tmpxililg8w']
[2023-12-27 17:54:25,377] {standard_task_runner.py:80} INFO - Job 97: Subtask extraer_data
[2023-12-27 17:54:25,624] {task_command.py:371} INFO - Running <TaskInstance: OpenWeather_ETL2.extraer_data manual__2023-12-27T17:54:20.503605+00:00 [running]> on host 44a36bef83ad
[2023-12-27 17:54:26,098] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-12-27 17:54:26,279] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Furtado_Julio
AIRFLOW_CTX_DAG_ID=OpenWeather_ETL2
AIRFLOW_CTX_TASK_ID=extraer_data
AIRFLOW_CTX_EXECUTION_DATE=2023-12-27T17:54:20.503605+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-27T17:54:20.503605+00:00
[2023-12-27 17:54:26,284] {logging_mixin.py:115} INFO - Adquiriendo data para la fecha: 2023-12-27 17
[2023-12-27 17:54:27,508] {logging_mixin.py:115} INFO - Datos adquiridos!
[2023-12-27 17:54:27,532] {python.py:173} INFO - Done. Returned value was: None
[2023-12-27 17:54:27,654] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=OpenWeather_ETL2, task_id=extraer_data, execution_date=20231227T175420, start_date=20231227T175425, end_date=20231227T175427
[2023-12-27 17:54:27,727] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-27 17:54:27,895] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
